### ISSUE TITLE
Status Change | Not Showing Pop-out

### ISSUE THREAD

**Issue by Aidan2662** [davart154/DTM-18#11]
![image](https://github.com/davart154/DTM-18/assets/111140166/5d1e4e6f-7c3b-4888-ba20-3849fa8adb72)
When I try clicking on my status, nothing happens. I can't change my status, only when the theme is off.
Is there an active solution for this?

**Comment by davart154** [comment]
Not sure why this happens, I'll look into it when I'm available though. Thanks for making me aware of this.

**Comment by Aidan2662** [comment]
Thanks for attempting a fix. I did see that it is showing an actual menu now, but it just gets more and more worse as you look up. Hopefully you get this figured out eventually!
![image](https://github.com/davart154/DTM-18/assets/111140166/e4bc9a35-e0bf-49b2-a4f3-cdd02c52a418)

**Comment by davart154** [comment]
Yeah this is because of simplified profiles exp. going on rollout. I have to redo that entire popout now.

**Comment by Aidan2662** [comment]
Oh nevermind, I realized that was a feature about a day ago. I'll give you a complement, this is the closest to Discord 2018, and you did an amazing job even if Discord broke some aspects of this theme. Not everything is perfect, but bugs sometimes need addressing.

**Comment by davart154** [comment]
Thanks for the kind words. I started this theme some time in 2021, when I had just started learning css, so I'll admit that some stuff isn't exactly written as well as it should be.

I recently started working on a rewrite of the theme so that I can make it cleaner and hopefully make it run a bit better. When I can, I'll make sure to try and fix some bugs in the current version as well, so if there's anything else that's really broken, feel free to let me know.
